Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "NetworkService",
```

Comment:
I thought the package has the "simple network service" name? Should the package of the name match the name of the repo?

Reply:
fixed